mibs/viginitos/VEL-MIB

```diff
@@ -0,0 +1,41 @@
+VEL-MIB DEFINITIONS ::= BEGIN
```

Comment:
The MIB directory should match the os name. (you have an extra i here)

Reply:
it was mistake, I changed it

Reply:
thank you :)